app/screens/post_options/post_options.tsx

```diff
@@ -73,64 +81,85 @@ const PostOptions = ({
+                    post={post}
+                    sourceScreen={sourceScreen}
+                />
                 }
                 {!isSystemPost &&
-                    <SaveOption
-                        isSaved={isSaved}
-                        postId={post.id}
-                    />
+                <SaveOption
+                    isSaved={isSaved}
+                    postId={post.id}
+                />
                 }
                 {Boolean(canCopyText && post.message) &&
-                    <CopyTextOption
-                        postMessage={post.message}
-                        sourceScreen={sourceScreen}
-                    />}
+                <CopyTextOption
+                    postMessage={post.message}
+                    sourceScreen={sourceScreen}
+                />}
                 {canPin &&
-                    <PinChannelOption
-                        isPostPinned={post.isPinned}
-                        postId={post.id}
-                    />
+                <PinChannelOption
+                    isPostPinned={post.isPinned}
+                    postId={post.id}
+                />
                 }
                 {canEdit &&
-                    <EditOption
-                        post={post}
-                        canDelete={canDelete}
-                    />
+                <EditOption
+                    post={post}
+                    canDelete={canDelete}
+                />
                 }
                 {canDelete &&
                 <DeletePostOption
                     combinedPost={combinedPost}
                     post={post}
                 />}
+                {shouldShowBindings &&
+                <AppBindingsPostOptions
+                    post={post}
+                    serverUrl={serverUrl}
+                    bindings={bindings}
+                />
+                }
             </>
         );
     };
 
     const additionalSnapPoints = 2;
+    const lowerSnapPoints = snapPoints + additionalSnapPoints;
+
+    let upperSnapPoints = lowerSnapPoints;
+    if (shouldShowBindings) {
+        upperSnapPoints += bindings.length;
+    }
 
     return (
         <BottomSheet
             renderContent={renderContent}
             closeButtonId={POST_OPTIONS_BUTTON}
             componentId={Screens.POST_OPTIONS}
-            initialSnapIndex={0}
-            snapPoints={[((snapPoints + additionalSnapPoints) * ITEM_HEIGHT), 10]}
+            initialSnapIndex={1}
+            snapPoints={[upperSnapPoints * ITEM_HEIGHT, lowerSnapPoints * ITEM_HEIGHT, 10]}
             testID='post_options'
         />
     );
 };
 
-export default PostOptions;
+type OwnProps = {
+    serverUrl: string;
+}
+
+const withBindings = withObservables([], (ownProps: OwnProps) => ({
+    bindings: AppsManager.observeBindings(ownProps.serverUrl, AppBindingLocations.POST_MENU_ITEM),
+}));
+
+export default React.memo(withServerUrl(withBindings(PostOptions)));
```

Comment:
the serverUrl could be added in `app/components/post_list/post/post.tsx?L#185` then you don't need to add the `withServerUrl` HOC